pages/_app.js

```diff
@@ -1,10 +1,19 @@
-import GlobalStyle from "../styles";
+import { SWRConfig } from "swr";
 
 export default function App({ Component, pageProps }) {
   return (
-    <>
-      <GlobalStyle />
```

Comment:
you will need the GlobalStyle in your project, I recommend you undo the removal of this line as well as of the import